lib/matplotlib/cbook.py

```diff
@@ -82,6 +84,64 @@ def _get_running_interactive_framework():
     return None
 
 
+@contextlib.contextmanager
+def _allow_interrupt(prepare_notifier, handle_sigint):
+    """
+    A context manager that allows terminating a plot by sending a SIGINT.  It
+    is necessary because the running backend prevents Python interpreter to
+    run and process signals (i.e., to raise KeyboardInterrupt exception).  To
```

Comment:
```suggestion
    is necessary because the running backend prevents the Python interpreter from
    running and processing signals (i.e., to raise KeyboardInterrupt exception).  To
```